lib/TypeNames/ModelToPTMLTypeHelpers.cpp

```diff
@@ -328,8 +328,12 @@ static void printDefinition(Logger<> &Log,
          << B.getKeyword(ptml::PTMLCBuilder::Keyword::Struct) << " "
          << B.getPackedAttribute() << " ";
 
-  if (ForEditing)
+  if (ForEditing) {
```

Comment:
Not just for editing, I'd like it in regular decompiled code too.

Reply:
Wait? Do you mean size? Or startAt? Or both?

Reply:
I meant `_CAN_CONTAIN_CODE`.

Reply:
Done!